frontend/public/components/edit-yaml.jsx

```diff
@@ -709,13 +709,20 @@ const EditYAML_ = connect(stateToProps)(
                         <Popover
                           aria-label="Shortcuts"
                           bodyContent={
-                            <ul>
-                              <li>Use Ctrl + Space to activate auto complete.</li>
-                              <li>
-                                Use {isMac ? 'Command' : 'Ctrl'} + Shift + O for document outlining.
-                              </li>
-                              <li>Hover over a property to view a description.</li>
-                            </ul>
+                            <ShortcutTable>
+                              <Shortcut macCtrl keyName="space">
+                                Use Ctrl + Space to activate auto complete.
+                              </Shortcut>
+                              <Shortcut ctrl shift keyName="o">
+                                Use Shift + {isMac ? 'Command' : 'Ctrl'} + O for document outlining.
```

Comment:
```suggestion
                                view document outline
```

Reply:
UX strings provided for dev-console topology began with upper case.
`View document outline`